### PR TITLE
daemonize the entire thing

### PR DESCRIPTION
**Pull request by ryzokuken**
I guess this was a really important thing for the daemon... to actually be one.

### CODE REVIEW

File: Makefile
Comment (ryzokuken) [inline Makefile:1]: `Makefiles` are fun, but I suppose since we're in a Node project it's just simpler to add new scripts to package.json instead...

**Review by ryzokuken — COMMENTED** [review]
Overall looks great to me @epilys. Do we absolutely need a Makefile?

**Comment by ryzokuken** [comment]
Oooh and I figured out the problem behind Prettier... I'm using it from my editor and you're probably doing the same or you've installed it globally... Either way, it's probably a good idea to add `prettier` in devDependencies and call the prettier from inside the project with or without a tiny prettier config. This would allow all prettier instances to align on the same set of rules.

File: Makefile
Comment (epilys) [reply Makefile:1]: old habits 😮‍💨

**Pull request merged**